{"uid":"b6ef86e0db1b33e5","name":"Валидация поля limit: 0","fullName":"tests.test_backend.board.board_validation_test#test_create_board_group_limit_validation","historyId":"3768af2b099f4d1541df6790e6d51761","time":{"start":1749744409018,"stop":1749744409317,"duration":299},"status":"passed","statusMessage":"XPASS Пограничное значение 0 требует уточнения требований","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"delete_output_dir","time":{"start":1749744399429,"stop":1749744399429,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"_verify_url","time":{"start":1749744399429,"stop":1749744399429,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"pytestconfig","time":{"start":1749744399429,"stop":1749744399429,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"base_url","time":{"start":1749744399429,"stop":1749744399429,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"owner_client","time":{"start":1749744399429,"stop":1749744400163,"duration":734},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_space","time":{"start":1749744400163,"stop":1749744401184,"duration":1021},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_project","time":{"start":1749744401184,"stop":1749744402659,"duration":1475},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_board","time":{"start":1749744405216,"stop":1749744405548,"duration":332},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"passed","statusMessage":"XPASS Пограничное значение 0 требует уточнения требований","steps":[{"name":"Проверка, что API вернул 200 при limit = 0","time":{"start":1749744409317,"stop":1749744409317,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":0,"hasContent":true},"afterStages":[{"name":"temp_space::0","time":{"start":1749744437355,"stop":1749744438003,"duration":648},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"@pytest.mark.xfail(reason='Пограничное значение 0 требует уточнения требований')"},{"name":"tag","value":"backend"},{"name":"parentSuite","value":"tests.test_backend.board"},{"name":"suite","value":"board_validation_test"},{"name":"host","value":"pkrvmxyh4eaekms"},{"name":"thread","value":"2515-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_backend.board.board_validation_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"expected_status","value":"200"},{"name":"limit","value":"0"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":7,"passed":3,"unknown":0,"total":10},"items":[{"uid":"41d95dce568d86cf","reportUrl":"https://vaizcom.github.io/autotests/124//#testresult/41d95dce568d86cf","status":"passed","statusDetails":"XPASS Пограничное значение 0 требует уточнения требований","time":{"start":1749744322288,"stop":1749744322552,"duration":264}},{"uid":"3e433bb215bbe938","reportUrl":"https://vaizcom.github.io/autotests/123//#testresult/3e433bb215bbe938","status":"skipped","statusDetails":"XFAIL Пограничное значение 0 требует уточнения требований\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f6ed7455e50>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749744003579,"stop":1749744003579,"duration":0}},{"uid":"3ae3e56cdeca8ee3","reportUrl":"https://vaizcom.github.io/autotests/122//#testresult/3ae3e56cdeca8ee3","status":"passed","statusDetails":"XPASS Пограничное значение 0 требует уточнения требований","time":{"start":1749743626235,"stop":1749743626518,"duration":283}},{"uid":"5116753782f99e2e","reportUrl":"https://vaizcom.github.io/autotests/119//#testresult/5116753782f99e2e","status":"skipped","statusDetails":"XFAIL Пограничное значение 0 требует уточнения требований\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fc7b5913490>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742820717,"stop":1749742820717,"duration":0}},{"uid":"2ddace66283bd832","reportUrl":"https://vaizcom.github.io/autotests/118//#testresult/2ddace66283bd832","status":"skipped","statusDetails":"XFAIL Пограничное значение 0 требует уточнения требований\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f9cb88f1f10>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742668118,"stop":1749742668118,"duration":0}},{"uid":"e227ed63fbaace8e","reportUrl":"https://vaizcom.github.io/autotests/117//#testresult/e227ed63fbaace8e","status":"skipped","statusDetails":"XFAIL Пограничное значение 0 требует уточнения требований\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f2b9f21f250>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742497920,"stop":1749742497920,"duration":0}},{"uid":"9ba6d4c2eb2a216f","reportUrl":"https://vaizcom.github.io/autotests/116//#testresult/9ba6d4c2eb2a216f","status":"skipped","statusDetails":"XFAIL Пограничное значение 0 требует уточнения требований\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7efcc7f78ad0>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749741234321,"stop":1749741234321,"duration":0}},{"uid":"eaa5a4753c476e86","reportUrl":"https://vaizcom.github.io/autotests/115//#testresult/eaa5a4753c476e86","status":"skipped","statusDetails":"XFAIL Пограничное значение 0 требует уточнения требований\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f28d1ddcd10>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749740826851,"stop":1749740826851,"duration":0}},{"uid":"e0e4dc48902cd874","reportUrl":"https://vaizcom.github.io/autotests/114//#testresult/e0e4dc48902cd874","status":"skipped","statusDetails":"XFAIL Пограничное значение 0 требует уточнения требований\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f271b110ad0>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749740653586,"stop":1749740653586,"duration":0}}]},"tags":["@pytest.mark.xfail(reason='Пограничное значение 0 требует уточнения требований')","backend"]},"source":"b6ef86e0db1b33e5.json","parameterValues":["200","0"]}